### DISCUSSION TITLE
Was statusCode ever available directly on error object?

### DISCUSSION THREAD

**Discussion by rightaway** [sindresorhus/got#1719] [Q&A] [answered]
I've been looking through some code a year or two old that got the status code through `e.statusCode` but it isn't working anymore. I need `e.response.statusCode` now.

I can't find any notice about such change in the change log. Could you point me to when that was changed? Was it even ever available directly on `e` or is this code I'm looking at not doing what I think it is?

**Comment by szmarczak** [comment, marked as answer]
`error.response.statusCode`

**Reply by szmarczak** [reply]
https://github.com/sindresorhus/got/releases/tag/v10.0.0

**Reply by szmarczak** [reply]
![image](https://user-images.githubusercontent.com/36894700/117834772-b7d56500-b277-11eb-880d-00b88d67648c.png)

**Comment by sindresorhus** [comment]
In [Got v10](https://github.com/sindresorhus/got/releases/tag/v10.0.0), we moved many error properties. There's a helpful diff there that shows the difference.
